16-draft.json

```diff
@@ -1307,6 +1307,146 @@
             ]
           },
           "minItems": 1
+        },
+        "particulate_matter": {
+          "description": "Concentration of particulate matter in the air.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "properties": {
+                "type": "object",
+                "properties": {
+                  "PM10": {
+                    "description": "Concentration of particulate matter with a diameter of 10µm or less.",
+                    "type": "object",
+                    "properties": {
+                      "value": {
+                        "description": "The measurement value",
+                        "type": "number",
+                        "minimum": 0
+                      },
+                      "unit": {
+                        "description": "The measurement unit",
+                        "type": "string",
+                        "enum": [
+                          "µg/m³"
+                        ]
+                      }
+                    },
+                    "required": [
+                      "value",
+                      "unit"
+                    ]
+                  },
+                  "PM4.25": {
+                    "description": "Concentration of particulate matter with a diameter of 4.25µm or less.",
```

Comment:
```suggestion
                  "PM4": {
                    "description": "Concentration of particulate matter with a diameter of 4µm or less.",
```

Reply:
I did some online research and found that "PM4.25" seems to be an industry standard. Are you sure "PM4" isn't just an abbreviation used by your sensor?